dev-python/m2crypto/m2crypto-0.31.0.ebuild

```diff
@@ -0,0 +1,52 @@
+# Copyright 1999-2018 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+
+PYTHON_COMPAT=( python2_7 python3_4 python3_5 python3_6)
+PYTHON_REQ_USE="threads(+)"
+
+inherit distutils-r1
+
+MY_PN="M2Crypto"
+
+DESCRIPTION="M2Crypto: A Python crypto and SSL toolkit"
+HOMEPAGE="https://gitlab.com/m2crypto/m2crypto https://pypi.org/project/M2Crypto/"
+SRC_URI="mirror://pypi/${MY_PN:0:1}/${MY_PN}/${MY_PN}-${PV}.tar.gz"
+
+LICENSE="MIT"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+
+# NOTE: Apparently nobody cares about libressl support, dropping support
+# IUSE="libressl"
+
+RDEPEND="
+	>=dev-libs/openssl-1.0.2:0=[-bindist(-)]
+	virtual/python-typing[${PYTHON_USEDEP}]
+"
+DEPEND="${RDEPEND}
+	>=dev-lang/swig-2.0.4:0
```

Comment:
Lowest version in portage tree is 2.0.9, and it doesn't appear to be slotted.

Reply:
Yup, correcting it (>2.0.4 was per upstream requirements).